src/api/schemas/extendedQueryNode.graphql

```diff
@@ -22,8 +29,6 @@ type Channel {
 
   avatarPhotoUrl: String
 
-  owner: Member!
```

Comment:
We still should have member reference in the `Channel` type. It will most likely be `ownerMember: Membership`